### PR TITLE
about page, link styling, home page text

### PR DESCRIPTION
**Pull request by zeldaskywalker**

### CODE REVIEW

File: mts-web-app/mts/static/css/site.css [hunk 1]
```diff
@@ -178,6 +208,11 @@ button {
     padding: 50px;
 }
 
+#issues-page-container {
+    padding: 50px;
+}
+9
```
Comment (zeldaskywalker) [inline mts-web-app/mts/static/css/site.css]: @eszypulski I think this is a typo.

File: mts-web-app/mts/templates/about.html [hunk 2]
```diff
@@ -27,23 +28,255 @@ <h2>ABOUT</h2>
...
+        <h3>DATABASE</h3>
+        <h4>POSTGRESQL | GEOJSON | PYTHON | GOOGLE SHEETS</h4>
+        <p>More Than Surviving is powered by a custom-built API that enables seamless integration between the site's
+            database and the data visualized in the interactive map and timeline. For accessibility and ease of future
+            updates, the site database can be updated through a Google Colab notebook that retrieves research data that
+            is put into Google Sheets. In the future, this will allow tribe members and other researchers to easily join
+            and expand the project.</p>
     </div>
 </div>
 {% endblock content %}
```
Comment (zeldaskywalker) [inline mts-web-app/mts/templates/about.html:282]: @eszypulski Add a newline at end of file.

File: mts-web-app/mts/templates/glossary.html [hunk 3]
```diff
@@ -1,5 +1,28 @@
...
+    </div>
+
+    <input type="checkbox" id="accordion3" />
+    <label for="accordion3">Header 3</label>
+    <div class="content">
+      <p>Content 3</p>
+    </div>
+  </div>
+</div>
 {% endblock content %}
```
Comment (zeldaskywalker) [inline mts-web-app/mts/templates/glossary.html]: @eszypulski Add newline at end of file.

File: mts-web-app/mts/static/css/site.css (hunk 1 above)
Comment (eszypulski) [reply mts-web-app/mts/static/css/site.css]: Fixed this, thanks

File: mts-web-app/mts/templates/about.html (hunk 2 above)
Comment (eszypulski) [reply mts-web-app/mts/templates/about.html:282]: Done, thanks!

File: mts-web-app/mts/templates/glossary.html (hunk 3 above)
Comment (eszypulski) [reply mts-web-app/mts/templates/glossary.html]: Done, thank you!

**Pull request merged**